pkg/datarepair/checker/checker.go

```diff
@@ -193,6 +193,9 @@ func (checker *Checker) updateSegmentStatus(ctx context.Context, pointer *pb.Poi
 
 	numHealthy := int32(len(pieces) - len(missingPieces))
 	redundancy := pointer.Remote.Redundancy
+	mon.IntVal("checker_segment_total_count").Observe(int64(len(pieces)))
```

Comment:
how would you confirm that these are actually being logged?

Reply:
It is not a log, it is actually a monkit statistic. So by going to `debugaddress/mon/stats`, you would see a bunch of details such as 
```
checker_segment_total_count sum 43919
checker_segment_total_count min	40
checker_segment_total_count avg	80
checker_segment_total_count max 100
```